src/org/ticket/model/dao/iGenericDAO.java

```diff
@@ -0,0 +1,11 @@
+package org.ticket.model.dao;
+
+import java.util.List;
+
+public interface iGenericDAO<T> {
+    void save(T object);
+    T search(Class objClass, String fieldName, String argument);
```

Comment:
Maybe return an `Optional` here, we can't assure it will find an element and return it.

Reply:
Done!